.github/workflows/erlang-tests.yml

```
@@ -0,0 +1,146 @@
+
+jobs:
+  test:
+    name: OTP ${{ matrix.otp }} – Ubuntu
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        otp: [24, 25, 26, 27]
+
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (3 > 2)

(empty-lines)